packages/search-ui-elasticsearch-connector/src/handlers/common.ts

```diff
@@ -20,7 +20,10 @@ export function fieldResponseMapper(
       };
     },
     {
-      id: { raw: item.id }
+      id: { raw: item.id },
+      _meta: {
+        id: item.rawHit?._id
```

Comment:
Looking at optional chaining... So it's possible for `_meta.id` to be undefined?
Will we hit the same error if so?